content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -1,8 +1,152 @@
 # DVC Files and Directories
 
 Once initialized in a <abbr>project</abbr>, DVC populates its installation
-directory (`.dvc/`) with the internal files and directories needed for DVC
-operation:
+directory (`.dvc/`) with the
+[internal directories and files](#internal-directories-and-files) needed for DVC
+operation.
+
+Additionally, there are two special kind of files created by certain
+[DVC commands](/doc/command-reference):
+
+- Files ending with the `.dvc` extension are placeholders to version data files
+  and directories. A <abbr>DVC project</abbr> usually has one
+  [`.dvc` file](#dvc-files) per large data file or dataset directory being
+  tracked.
+- The [`dvc.yaml` file](#dvcyaml-file) or _pipeline(s) file_ specifies stages
+  that form the pipeline(s) of a project, and their connections (_dependency
+  graph_ or DAG).
+
+Both use human-friendly YAML schemas, described below. We encourage you to get
+familiar with them so you may edit them freely, as needed. Both type of files
+should be versioned with Git (for Git-enabled <abbr>repositories</abbr>).
+
+## .dvc file
+
+When you add a file or directory to a <abbr>DVC project</abbr> with `dvc add` or
+`dvc import`, a `.dvc` file is created based on the data file name (e.g.
+`data.xml.dvc`). These files contain the information needed to track the data
+with DVC.
+
+They use a simple [YAML](https://yaml.org/) format, meant to be easy to read,
+edit, or even created manually by users. Here is a full sample:
+
+```yaml
+outs:
+  - md5: a304afb96060aad90176268345e10355
+    path: data.xml
+
+# Comments and user metadata are supported.
+meta:
+  name: 'John Doe'
+  email: john@doe.com
+```
+
+`.dvc` files can contain the following fields:
+
+- `outs`: List of <abbr>output</abbr> entries for this `.dvc` file. Typically
+  there is only one (but several can be added manually).
+- `deps` (optional): List of <abbr>dependency</abbr> entries for this stage,
+  only present when `dvc import` and `dvc import-url` are used. Typically there
+  is only one (but several can be added manually).
+- `meta` (optional): Arbitrary metadata can be added manually with this field.
+  Any YAML contents is supported. `meta` contents are ignored by DVC, but they
+  can be meaningful for user processes that read `.dvc` files.
+
+An _output entry_ can consist of these fields:
+
+- `md5` (optional): Hash value for the output file
```

Comment:
it's not optional I guess ... it depends on the definition of optional of course ... that's why let's just remove them for now everywhere

Reply:
Yes, all removed now. I failed to push my last commit this morning sorry 🤦